pkg/karmadactl/util/pubkeypin/pubkeypin.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
The reason why you copy the files is due to the concern of huge dependencies, right?

Can we put it under `pkg/util/lifted`?

Reply:
> The reason why you copy the files is due to the concern of huge dependencies, right?

Yes, `kubeadm` doesn't provide a standard dependent package, only can import it with `k8s.io/kubernetes` that is not recommended.

> Can we put it under pkg/util/lifted?

Sure, I only think the work of putting it to `pkg/util/lifted` is a lit redious~, Hhh.

Reply:
You can move the whole package there.